### ISSUE TITLE
transparent default template

### ISSUE THREAD

**Issue by mkoohafkan** [rstudio/bookdown#413]
See the discussion in the comments of this [stackoverflow question](https://stackoverflow.com/questions/44770287/adding-natbib-options-for-pdf-output)

Setting `template: default` in the bookdown YAML for LaTeX/PDF output does not have the same result as specifying the default template provided by rmarkdown (which is only apparent if you follow the `pdf_document` argument `base_format` to the source code of `rmarkdown::pdf_document`). It appears that `geometry: "margin=1in"` gets set internally when `template: default` is specified, but is not set when you specify the path to the default template file itself. This is confusing, and can lead people (like me!) astray when they start trying to customize a template and are unable to reproduce the default look of the pdf document. It would be good to remove this inconsistent behavior, or at least document this somewhere,.

On a more general note, I think it would be helpful for bookdown include the default LaTeX template with the package (as it does with the HTML templates), even if it is redundant. It would make it easier for users to access the default template and build off it, and would allow bookdown to provide its own default template in the future that might be more appropriate for books than the default pandoc template. Alternatively, an explicit statement of where to find the "default" template and would be helpful.

**Comment by jemus42** [comment]
I agree on the included LaTeX template. At least a simple preamble that includes any custom environments like those for the `rmdnote` blocks etc.  

Also, rendering a bookdown book to PDF has been a world of pain for me in general, and I assume a decently customized template could fix a lot of the issues I've had (I've since given up and started converting the epub output to PDF via `bookdown::calibre`, which works wonderfully for my case)
